webpack.config.js

```diff
@@ -41,9 +41,10 @@ module.exports = {
           transpileOnly: true,
           compilerOptions: { target: "esnext" }
         }
-      }
+      },
     ]
   },
+  target: 'node',
```

Comment:
Do we only targeting node here and not web or am i missing something here ?

Reply:
updated